Patch to make transcode use =lzo-2* instead of old lzo-1*.
Contributed by Daniel Black <dragonheart@gentoo.org>

See Gentoo Bug #112382

diff -ru transcode-1.0.2.orig/configure.in transcode-1.0.2/configure.in
--- transcode-1.0.2.orig/configure.in	2005-10-30 15:57:45.000000000 +1100
+++ transcode-1.0.2/configure.in	2005-11-13 20:13:26.000000000 +1100
@@ -1018,7 +1018,7 @@
 dnl
 dnl LZO
 dnl
-TC_PKG_CHECK(lzo, no, LZO, no, [lzo1x.h], lzo, lzo_version, lzo,
+TC_PKG_CHECK(lzo, no, LZO, no, [lzo1x.h], lzo2, lzo_version, lzo2,
  [http://www.oberhumer.com/opensource/lzo/])
 TC_PKG_HAVE(lzo, LZO)
 
diff -ru transcode-1.0.2.orig/export/export_lzo.c transcode-1.0.2/export/export_lzo.c
--- transcode-1.0.2.orig/export/export_lzo.c	2005-07-04 17:09:31.000000000 +1000
+++ transcode-1.0.2/export/export_lzo.c	2005-11-13 20:09:26.000000000 +1100
@@ -84,8 +84,8 @@
 	return(TC_EXPORT_ERROR); 
       }
 
-      wrkmem = (lzo_bytep) lzo_malloc(LZO1X_1_MEM_COMPRESS);
-      out = (lzo_bytep) lzo_malloc(vob->ex_v_height*vob->ex_v_width*3*2);
+      wrkmem = (lzo_bytep) malloc(LZO1X_1_MEM_COMPRESS);
+      out = (lzo_bytep) malloc(vob->ex_v_height*vob->ex_v_width*3*2);
 
       if (wrkmem == NULL || out == NULL) {
 	printf("[%s] out of memory\n", MOD_NAME);
@@ -270,8 +270,8 @@
   
   if(param->flag == TC_VIDEO) {
 
-    lzo_free(wrkmem);
-    lzo_free(out);
+    free(wrkmem);
+    free(out);
     
     return(0);
   }
diff -ru transcode-1.0.2.orig/import/decode_lzo.c transcode-1.0.2/import/decode_lzo.c
--- transcode-1.0.2.orig/import/decode_lzo.c	2005-07-04 17:09:33.000000000 +1000
+++ transcode-1.0.2/import/decode_lzo.c	2005-11-13 20:09:26.000000000 +1100
@@ -65,9 +65,9 @@
       goto decoder_error;
     }
 
-    wrkmem = (lzo_bytep) lzo_malloc(LZO1X_1_MEM_COMPRESS);
-    out = (lzo_bytep) lzo_malloc(BUFFER_SIZE);
-    inbuf = (lzo_bytep) lzo_malloc(BUFFER_SIZE);
+    wrkmem = (lzo_bytep) malloc(LZO1X_1_MEM_COMPRESS);
+    out = (lzo_bytep) malloc(BUFFER_SIZE);
+    inbuf = (lzo_bytep) malloc(BUFFER_SIZE);
     
     if (wrkmem == NULL || out == NULL) {
       fprintf(stderr, "[%s] out of memory\n", MOD_NAME);
diff -ru transcode-1.0.2.orig/import/import_lzo.c transcode-1.0.2/import/import_lzo.c
--- transcode-1.0.2.orig/import/import_lzo.c	2005-07-04 17:09:33.000000000 +1000
+++ transcode-1.0.2/import/import_lzo.c	2005-11-13 20:09:26.000000000 +1100
@@ -117,8 +117,8 @@
       return(TC_IMPORT_ERROR); 
     }
 
-    wrkmem = (lzo_bytep) lzo_malloc(LZO1X_1_MEM_COMPRESS);
-    out = (lzo_bytep) lzo_malloc(BUFFER_SIZE);
+    wrkmem = (lzo_bytep) malloc(LZO1X_1_MEM_COMPRESS);
+    out = (lzo_bytep) malloc(BUFFER_SIZE);
 
     if (wrkmem == NULL || out == NULL) {
       printf("[%s] out of memory\n", MOD_NAME);
@@ -236,8 +236,8 @@
 
   if(param->flag == TC_VIDEO) {
 
-    lzo_free(wrkmem);
-    lzo_free(out);
+    free(wrkmem);
+    free(out);
 
     if(avifile2!=NULL) {
       AVI_close(avifile2);
